pkg/authorization/api/register.go

```diff
@@ -89,16 +88,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 
 		&RoleBindingRestriction{},
 		&RoleBindingRestrictionList{},
-	}
-	scheme.AddKnownTypes(SchemeGroupVersion,
-		append(types,
-			&metav1.Status{}, // TODO: revisit in 1.6 when Status is actually registered as unversioned
```

Comment:
It's unversioned and registered as and in core v1. We had that discussion before. I would like to see it outside of pkg/api as well. But not in the rebase.